data/rulesets.ts

```diff
@@ -38,7 +38,7 @@ export const Formats: {[k: string]: FormatsData} = {
 			'Meltan', 'Melmetal', 'Zacian', 'Zamazenta', 'Eternatus',
 		],
 		onValidateSet(set, format) {
-			if (this.gen < 7 && toID(set.item) === 'souldew') {
+			if (this.gen < 7 && this.dex.toID(set.item) === 'souldew') {
```

Comment:
We should probably add `toID` to team validator contexts, so we can use `this.toID` in rulesets.